File: template/template.go
package main

import "fmt"
import "os"
import "strconv"
import "time"

func f() int {
    // PROBLEM SOLUTION GOES HERE
    return 0
}

func main() {
    if len(os.Args) == 1 {
        fmt.Println(f())
    } else if len(os.Args) == 2 {
        iters, _ := strconv.ParseInt(os.Args[1], 10, 64)
        start := time.Now()
        for i := int64(0); i < iters; i++ {
            f()
        }
        elapsed := time.Now().Sub(start)

        fmt.Println(elapsed.Nanoseconds() / iters)
    }

}

File: template/template.js
function f() {
    // PROBLEM SOLUTION GOES HERE
    return 0;
}

function toNs(ts) {
    return ts.sec * 1000000000 + ts.nsec;
}

var clock = require('posix-clock');

if (process.argv.length === 3) {
    var i;
    var iters = parseInt(process.argv[2], 10);

    var start = clock.gettime(clock.MONOTONIC);
    for (i = 0; i < iters; i++) {
        f();
    }
    var end = clock.gettime(clock.MONOTONIC);

    console.log(toNs(end) - toNs(start));
} else {
    console.log(f());
}
